jxls-poi/src/test/java/org/jxls3/EachTest.java

```diff
@@ -0,0 +1,218 @@
+
+        // Test
+        Jxls3Tester tester = Jxls3Tester.xlsx(getClass(), "orderByIgnoreCase");
+        tester.test(data, JxlsPoiTemplateFillerBuilder.newInstance());
+        
+        // Verify
+        try (TestWorkbook w = tester.getWorkbook()) {
+            w.selectSheet("Employees");
+            assertEquals("A", w.getCellValueAsString(2, 1)); 
+            assertEquals("i", w.getCellValueAsString(3, 1)); 
+            assertEquals("Z", w.getCellValueAsString(4, 1)); 
+        }
+    }
+
+	// This is also a direction=RIGHT test.
+    @Test
+    public void ignoreProps() throws ParseException {
+    	// Prepare
+		List<EmployeeWithDepartments> employees = new ArrayList<>();
+		employees.add(new EmployeeWithDepartments("Kylie", "1970-Jul-10", 2100, "Main"));
+		employees.add(new EmployeeWithDepartments("Mariah", "1943-Jan-01", 0));
+		employees.add(new EmployeeWithDepartments("Sascha", "1965-Mar-19", 600, "D", "E", "F", "G", "H"));
+		Map<String, Object> data = new HashMap<>();
+        data.put("employees", employees);
+        
+        // Test
+        Jxls3Tester tester = Jxls3Tester.xlsx(getClass(), "ignoreProps");
+        tester.test(data, JxlsPoiTemplateFillerBuilder.newInstance()
+        		.withIgnoreColumnProps(true).withIgnoreRowProps(true)); // Jxls doesn't change row heights and column widths.
+        
+        // Verify
+        try (TestWorkbook w = tester.getWorkbook()) {
+            w.selectSheet("Employees");
+            assertEquals("Sascha", w.getCellValueAsString(4, 1));
+            assertEquals("direction=RIGHT does not work", "H", w.getCellValueAsString(4, 8));
+            assertFalse(w.getColumnWidth(4) == w.getColumnWidth(5));
+            assertFalse(w.getColumnWidth(5) == w.getColumnWidth(6));
+            assertFalse(w.getRowHeight(2) == w.getRowHeight(3));
+        }
+        
+        // Test
+        tester = Jxls3Tester.xlsx(getClass(), "ignoreProps");
+        tester.test(data, JxlsPoiTemplateFillerBuilder.newInstance()
+        		.withIgnoreColumnProps(false).withIgnoreRowProps(false)); // Jxls creates rows and columns with the same width/height. (default)
+        
+        // Verify
+        try (TestWorkbook w = tester.getWorkbook()) {
+            w.selectSheet("Employees");
+            assertTrue(w.getColumnWidth(4) == w.getColumnWidth(5));
+            assertTrue(w.getColumnWidth(5) == w.getColumnWidth(6));
+            assertTrue(w.getRowHeight(2) == w.getRowHeight(3));
+        }
+    }
+    
+    public static class EmployeeWithDepartments extends Employee {
+        private static final SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MMM-dd", Locale.US);
+    	private final List<String> departments = new ArrayList<>();
+        
+    	public EmployeeWithDepartments(String name, String birthDate, double payment, String ...departments) throws ParseException {
+        	super(name, dateFormat.parse(birthDate), payment, 0d);
+        	for (String i : departments) {
```

Comment:
Perhaps we can use `this.departments.addAll(Arrays.asList(departments));` instead of the `for` loop?